ansible/roles/ocp4-workload-enable-lets-encrypt-certificates/files/deploy_certs.yml

```diff
@@ -52,7 +54,7 @@
   - name: Create new Ingress Controller Certificate
     k8s:
       state: present
-      definition: "{{ lookup('template', './router-certs.j2' ) | from_yaml }}"
+      definition: "{{ lookup('template', './templates/router-certs.j2' ) | from_yaml }}"
```

Comment:
On a different line, the upstream team removed the "templates" sub path. We should check if they made a change that impacts this. https://github.com/RedHatTraining/agnosticd/commit/6eca6903e03e0f57e7f6c50f34ab863a85d5b053#diff-ec046d14576b79d8c80b28e4bc25e8a1